e2e/transactions/transactions_test.go

```diff
@@ -314,6 +314,89 @@ func (s *TransactionsTestSuite) TestSendEtherTx() {
 	s.Zero(s.Backend.TxQueueManager().TransactionQueue().Count(), "tx queue must be empty at this point")
 }
 
+func (s *TransactionsTestSuite) TestSendEtherOnStatusChainTx() {
+
```

Comment:
This new line is not needed.

Reply:
Do we have a style guide section on that? Should I really fix it?

Reply:
Very minor. I just noticed it and though it's a typo or copy&paste. Generally, we don't put a new line at the beginning of a function body.

Reply:
Ok, thanks. It would be great if it was stated in Style Guide so that the other contributor kept it in mind as well.

Reply:
We don't add blank lines after a function declaration, I'll add it to our upcoming style guide. Please, fix.

Reply:
We don't add blank lines after a function declaration, I'll add it to our upcoming style guide. Please, fix.